geonode/layers/templates/layers/layer_geoext_map.html

```diff
@@ -45,6 +45,13 @@
 
                 listeners: {
                     "ready": function() {
+                        app.mapPanel.map.getMaxExtent = function() {
+                            return new OpenLayers.Bounds(-80150033.36,-80150033.36,80150033.36,80150033.36);
```

Comment:
should this be hardcoded?

Reply:
Could be improved of course. I just keep the original version and applied the minimal set of changes to let it working as espected.

Reply:
ok thanks